settings.local.yml

```diff
@@ -0,0 +1,2 @@
+saml:
```

Comment:
this file shouldn't be checked in

Reply:
ahhhhh that's it. totally forgot to ignore that one 🤦. already went ahead and created a clean PR over [here](https://github.com/department-of-veterans-affairs/vets-api/pull/4969).